client/reader/comments/index.jsx

```diff
@@ -222,6 +222,10 @@ var PostCommentList = React.createClass( {
 		CommentStore.on( 'add', this._onAdd );
 	},
 
+	componentDidUpdate: function() {
+		this.props.commentsUpdated();
```

Comment:
When we're passing a method as a prop to respond to an event, we've generally used naming like `onNounVerb`.

Can we use `onCommentsUpdate` rather than `commentsUpdated`?